applications/visualizer/backend/api/api.py

```diff
@@ -237,6 +245,23 @@ def get_connections(
 #     )
 
 
+## Ingestion
+
+
+@api.get("/populate_db", auth=basic_auth_superuser, tags=["ingestion"])
+@with_stdout_streaming
+def populate_db(request):
```

Comment:
I wonder if the endpoint shouldn't be async, perhaps it will have an impact on the decorator and on the final result?